client/components/errorLabel/errorLabel.css

```diff
@@ -1,4 +1,4 @@
-.button-error {
+.error {
```

Comment:
You should use color variables defined in color.css instead of color codes.
If the desired color is missing, you must add it in color.css